### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by KCNilssen**
### Why

Fixes and updates some docstrings and adds addition functionality to a few models.

### Tests

Passes test suite

### Risk and impact

Medium

### CODE REVIEW

File: mlbstatsapi/models/game/livedata/boxscore/attributes.py [hunk 1]
```diff
@@ -39,6 +40,73 @@ def __post_init__(self):
...
+    issubstitute : bool
+        Whether the player is a substitute.
+    """
+    iscurrentbatter: bool
+    iscurrentpitcher: bool
+    isonbench: bool
+    issubstitute: bool
+
+@dataclass
+class PlayersDictPerson:
```
Comment (Mattsface) [inline mlbstatsapi/models/game/livedata/boxscore/attributes.py:65]: Can't we just update the Player class in people for this?
 
 What's the endpoint look like?

Comment (KCNilssen) [reply mlbstatsapi/models/game/livedata/boxscore/attributes.py:65]: Nope, Its not a person endoing.

Here is one entry in the list:

```"ID623457": {   
    "person": {   
        "id": 623457,   
        "fullName": "Matt Rose",   
        "link": "/api/v1/people/623457"   
    },   
    "jerseyNumber": "93",   
    "position": {   
        "code": "3",   
        "name": "First Base",   
        "type": "Infielder",   
        "abbreviation": "1B"   
    },   
    "status": {   
        "code": "MIN",   
        "description": "Minor League Contract"   
    },   
    "parentTeamId": 145,   
    "stats": {   
        "batting": {},   
        "pitching": {},   
        "fielding": {}   
    },   
    "seasonStats": {   
        "batting": {   
            "gamesPlayed": 2,   
            "flyOuts": 0,   
            "groundOuts": 1,   
            "runs": 0,   
            "doubles": 1,   
            "triples": 0,   
            "homeRuns": 0,   
            "strikeOuts": 1,   
            "baseOnBalls": 0,   
            "intentionalWalks": 0,   
            "hits": 1,   
            "hitByPitch": 0,   
            "avg": ".333",   
            "atBats": 3,   
            "obp": ".333",   
            "slg": ".667",   
            "ops": "1.000",   
            "caughtStealing": 0,   
            "stolenBases": 0,   
            "stolenBasePercentage": ".---",   
            "groundIntoDoublePlay": 1,   
            "groundIntoTriplePlay": 0,   
            "plateAppearances": 3,   
            "totalBases": 2,   
            "rbi": 0,   
            "leftOnBase": 2,   
            "sacBunts": 0,   
            "sacFlies": 0,   
            "babip": ".500",   
            "catchersInterference": 0,   
            "pickoffs": 0,   
            "atBatsPerHomeRun": "-.--"   
        },   
        "pitching": {   
            "gamesPlayed": 0,   
            "gamesStarted": 0,   
            "groundOuts": 0,   
            "airOuts": 0,   
            "runs": 0,   
            "doubles": 0,   
            "triples": 0,   
            "homeRuns": 0,   
            "strikeOuts": 0,   
            "baseOnBalls": 0,   
            "intentionalWalks": 0,   
            "hits": 0,   
            "hitByPitch": 0,   
            "atBats": 0,   
            "obp": ".000",   
            "caughtStealing": 0,   
            "stolenBases": 0,   
            "stolenBasePercentage": ".---",   
            "numberOfPitches": 0,   
            "era": "-.--",   
            "inningsPitched": "0.0",   
            "wins": 0,   
            "losses": 0,   
            "saves": 0,   
            "saveOpportunities": 0,   
            "holds": 0,   
            "blownSaves": 0,   
            "earnedRuns": 0,   
            "whip": "-",   
            "battersFaced": 0,   
            "outs": 0,   
            "gamesPitched": 0,   
            "completeGames": 0,   
            "shutouts": 0,   
            "balls": 0,   
            "strikes": 0,   
            "strikePercentage": "-.--",   
            "hitBatsmen": 0,   
            "balks": 0,   
            "wildPitches": 0,   
            "pickoffs": 0,   
            "groundOutsToAirouts": "-.--",   
            "rbi": 0,   
            "winPercentage": ".---",   
            "pitchesPerInning": "-.--",   
            "gamesFinished": 0,   
            "strikeoutWalkRatio": "-.--",   
            "strikeoutsPer9Inn": "-.--",   
            "walksPer9Inn": "-.--",   
            "hitsPer9Inn": "-.--",   
            "runsScoredPer9": "-.--",   
            "homeRunsPer9": "-.--",   
            "inheritedRunners": 0,   
            "inheritedRunnersScored": 0,   
            "catchersInterference": 0,   
            "sacBunts": 0,   
            "sacFlies": 0,   
            "passedBall": 0   
        },   
        "fielding": {   
            "caughtStealing": 0,   
            "stolenBases": 0,   
            "stolenBasePercentage": ".---",   
            "assists": 0,   
            "putOuts": 0,   
            "errors": 0,   
            "chances": 0,   
            "fielding": ".000",   
            "passedBall": 0,   
            "pickoffs": 0   
        }   
    },   
    "gameStatus": {   
        "isCurrentBatter": false,   
        "isCurrentPitcher": false,   
        "isOnBench": true,   
        "isSubstitute": false   
    }   
},   ```

Comment (KCNilssen) [reply mlbstatsapi/models/game/livedata/boxscore/attributes.py:65]: Might be able to use mlb_module.merge_keys to make it use a person object but the issue still resides with gameStatus and the two stats

Comment (Mattsface) [reply mlbstatsapi/models/game/livedata/boxscore/attributes.py:65]: What is the url?

Comment (KCNilssen) [reply mlbstatsapi/models/game/livedata/boxscore/attributes.py:65]: https://statsapi.mlb.com/api/v1.1/game/534196/feed/live

Check,

Livedata -> boxscore -> teams -> away -> players

Comment (Mattsface) [reply mlbstatsapi/models/game/livedata/boxscore/attributes.py:65]: Okay I see what you are talking about.

**Review by Mattsface — APPROVED** [review]
I bet we can use that stat objects I have for the stats

**Pull request merged**